FileCloner/Models/SummaryGenerator.cs

```diff
@@ -130,11 +130,11 @@ private static void UpdateEntryWithNewData(JsonElement element, string relativeP
             if (element.TryGetProperty("LAST_MODIFIED", out JsonElement newLastModifiedProp) && existingMetadata.ContainsKey("LAST_MODIFIED"))
             {
                 DateTime newTimestamp = DateTime.Parse(newLastModifiedProp.GetString()!, null, DateTimeStyles.AdjustToUniversal);
-                DateTime existingTimestamp = DateTime.Parse(existingMetadata["LAST_MODIFIED"].ToString(), null, DateTimeStyles.AdjustToUniversal);
+                DateTime existingTimestamp = DateTime.Parse(existingMetadata["LAST_MODIFIED"].ToString() ?? "12:00:00", null, DateTimeStyles.AdjustToUniversal);

```

Comment:
Ths line is not clear could you provide a comment ?
PS: I know that I wrote this file :)

Reply:
Will do that and push. Should I give the default value as "UNKNOWN"  in case the last modified time stamp does not exist?

Reply:
Yes